Limit Clevo M720SR to C2, going to C3 makes the machine freeze. This
needs more investigation, sis chipset/bios problem may be?

Signed-off-by: Herton Ronaldo Krzesinski <herton@mandriva.com>

---
 drivers/acpi/processor_idle.c |    3 +++
 1 file changed, 3 insertions(+)

--- linux-2.6.33-rc8-git4-t2/drivers/acpi/processor_idle.c.orig	2010-02-20 12:45:35.000000000 +0200
+++ linux-2.6.33-rc8-git4-t2/drivers/acpi/processor_idle.c	2010-02-20 12:53:39.679099592 +0200
@@ -110,6 +110,9 @@ static struct dmi_system_id __cpuinitdat
 	  DMI_MATCH(DMI_BIOS_VENDOR,"Phoenix Technologies LTD"),
 	  DMI_MATCH(DMI_BIOS_VERSION,"SHE845M0.86C.0013.D.0302131307")},
 	 (void *)2},
+ 	{ set_max_cstate, "Clevo M720SR", {
+ 	  DMI_MATCH(DMI_BIOS_VENDOR,"Phoenix Technologies LTD"),
+ 	  DMI_MATCH(DMI_PRODUCT_NAME,"M720SR")}, (void *)2},
 	{ set_max_cstate, "Pavilion zv5000", {
 	  DMI_MATCH(DMI_SYS_VENDOR, "Hewlett-Packard"),
 	  DMI_MATCH(DMI_PRODUCT_NAME,"Pavilion zv5000 (DS502A#ABA)")},
